include/nbl/video/utilities/IGPUObjectFromAssetConverter.h

```diff
@@ -1001,9 +1001,8 @@ auto IGPUObjectFromAssetConverter::create(const asset::ICPUImage** const _begin,
 
             if (needToCompMipsForThisImg(cpuimg))
             {
-                // Todo(achal): Get format props from the physical device and check
-                // if vkCmdBlitImage can be used, assert for now until we do polyphase
-                // in compute
+                const auto& formatProps = _params.device->getPhysicalDevice()->getFormatProperties(cpuimg->getCreationParameters().format);
+                assert((formatProps.optimalTilingFeatures& asset::EFF_SAMPLED_IMAGE_FILTER_LINEAR_BIT).value); // for blits, can lift are polyphase compute
```

Comment:
what's the difference between `EFF_SAMPLED_IMAGE_BIT` and `EFF_SAMPLED_IMAGE_FILTER_LINEAR_BIT `


btw the format should support:
- TRANSFER DST
then to perform polyphase in compute:
- SAMPLED IMAGE
- STORAGE IMAGE